### PR TITLE
Add draft version of schedule

### LINKED ISSUE #1: Draft version of schedule
A draft version appropriate for linking in initial HubSpot emails that go out to participants (e.g., does not have to be complete with links to the correct release, etc.)

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
But future proof some links with `{{site.release_tag}}` (currently `master`; #3). This version of the schedule is intended to give folks an idea of what the structure and content of the workshop, but is subject to change with the addition of daily exercises.

Closes #1.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
From team meeting: move ggplot2 to day 2

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -5,13 +5,36 @@ nav_title: Schedule
...
+| 3:15        | [Bulk RNA-seq Exercise](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/06-bulk_rnaseq_exercise.Rmd) |
+| 3:30        | [Consultation session](workshop-structure.md#consultation-sessions) begins |
 | 5:00        | End             |
-| **Day 2**   | **Date**  | 
-| **Day 3**   | **Date**  |               
-| **Day 4**   | **Date**  | 
-| **Day 5**   | **Date**  |     
+| **Day 5**   | **2020-06-36** <br> Consultation and Presentations |
+| 12:00       | [Consultation session](workshop-structure.md#consultation-sessions) begins |
+| 2:30        | [Presentations begin](workshop-structure.md#) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md]:
```suggestion
| 2:30        | [Presentations begin](workshop-structure.md#presentations) |
```

File: workshop/SCHEDULE.md [hunk 2]
```diff
@@ -5,13 +5,36 @@ nav_title: Schedule
...
+| 12:00       | Bulk RNA-seq I: [QC, trim, and quantification with Salmon](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/01-qc_trim_quant.md)            |
+| 1:30       | Bulk RNA-seq II: [Gene-Level Summaries with tximport](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/02-gastric_cancer_tximport.nb.html) |
+| 2:30        | Bulk RNA-seq III: [Exploratory Analyses](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/03-gastric_cancer_exploratory.nb.html) |   
+| 3:30        | [Consultation session](workshop-structure.md#consultation-sessions) begins |
+| 5:00        | End             |        
+| **Day 4**   | **2020-06-25** <br> [Introduction to Bulk RNA-seq](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/README.md) | 
+| 12:00       | Exercise: [Neuroblastoma Cell Line tximport](https://github.com/AlexsLemonade/training-modules/blob/master/RNA-seq/04-nb_cell_line_tximport.md) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md]: Missed one!
```suggestion
| 12:00       | Exercise: [Neuroblastoma Cell Line tximport](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/04-nb_cell_line_tximport.md) |
```

**Review by jashapiro — COMMENTED** [review]
A couple little link fixes, but looks good overall.
Tested local jekyll build.

**Review by jashapiro — APPROVED** [review]
The links they work.

**Pull request merged**